alacritty/src/event.rs

```diff
@@ -299,10 +299,13 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
     }
 
     fn spawn_new_instance(&mut self) {
-        let alacritty = env::args().next().unwrap();
+        let mut env_args = env::args();
+        let alacritty = env_args.next().unwrap();
+
+        let working_directory_set;
```

Comment:
The changes look fine, however I wonder whether we can return set working directory by returning a tuple in a place where you set `args`. so it'll be `let (mut args, working_directory_set) = { (args, false) };` That should save a bunch of lines.

Reply:
Actually there's a far simpler solution, I was just being silly.